extensions/agroal/runtime/src/main/java/io/quarkus/agroal/runtime/DataSources.java

```diff
@@ -64,6 +64,14 @@
 public class DataSources {
 
     private static final Logger log = Logger.getLogger(DataSources.class.getName());
+    public static final String DEPRECATED_URL_PROPERTY_NAME_ERROR_MESSAGE_FORMAT = "`quarkus.datasource.%s.url` is deprecated and will be removed in a future version - it is "
+            +
+            "recommended to switch to `quarkus.datasource.%s.jdbc.url`. See https://quarkus.io/guides/datasource";
+    public static final String DEPRECATED_URL_PROPERTY_ERROR_MESSAGE = "`quarkus.datasource.url` is deprecated and will be removed in a future version - it is "
+            +
+            "recommended to switch to `quarkus.datasource.jdbc.url`. See https://quarkus.io/guides/datasource";
+    public static final String DATASOURCE_JDBC_URL_HAS_NOT_BEEN_DEFINED = "`quarkus.datasource.jdbc.url` has not been defined";
+    public static final String DATASOURCE_JDBC_S_URL_HAS_NOT_BEEN_DEFINED_MESSAGE_FORMAT = "`quarkus.datasource.jdbc.%s.url` has not been defined";
```

Comment:
I would get rid of these constants, there's no value for them IMHO.

Reply:
fixed